lib/rules/no-global-jquery.js

```diff
@@ -48,8 +52,18 @@ module.exports = {
         }
       },
 
+      VariableDeclarator(node) {
+        if (isVariableAssigmentGlobalJquery(node)) {
+          context.report(node, MESSAGE);
+        } else {
+          const varName = get(node, 'id.name');
+          emberJqueryVariables.push(varName);
+        }
+      },
+
       CallExpression(node) {
-        if (!isDestructured(node) && isGlobalJquery(node)) {
+        const calleeName = get(node, 'callee.name');
+        if (!isDestructured(node) && isGlobalJquery(node) && !emberJqueryVariables.includes(calleeName)) {
```

Comment:
Just to clarify, the point of `emberJqueryVariables` is to avoid issuing an error when a local variable was used (and therefore it no longer falls under this particular rule).  Is that correct?

Reply:
`emberJqueryVariables` is meant to collection things like `const $ = Ember.$`: variables with names that would otherwise trigger the rule (`$`, `jQuery`), but are actually safe.

Reply:
perfect, thanks for confirming